runtime/mainnet/Cargo.toml

```diff
@@ -0,0 +1,198 @@
+[package]
+name = "mainnet-runtime"
+version = "0.1.0"
+authors.workspace = true
+description = "A new Cumulus FRAME-based Substrate Runtime, ready for hacking together a parachain."
```

Comment:
Same, we can update to the metaquity-network details.

Reply:
Updated